applets/007-kill/applet.py

```diff
@@ -0,0 +1,47 @@
+
+def clean_directory(x, host_ip, host_user):
+
```

Comment:
no empty lines here. After function declaration the code should begin instantly. Linebreaks probably only to divide larger functional groups. Right?